### PR TITLE
refine the error mesaage of fill_constant and eye 

### PR DESCRIPTION
**Pull request by wangchaochaohu**
### PR types

Others
### PR changes

OPs
### Describe

refine the error message of fill_constant and eye

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -8256,6 +8256,7 @@ def gather(input, index, overwrite=True):
 
         .. code-block:: python
 
+            import paddle
```
Comment (willthefrog) [inline python/paddle/fluid/layers/nn.py]: unrelated?

File: python/paddle/tensor/manipulation.py [hunk 2]
```diff
@@ -474,9 +474,6 @@ def stack(x, axis=0, name=None):
 
 def split(x, num_or_sections, axis=0, name=None):
     """
-	:alias_main: paddle.split
```
Comment (willthefrog) [inline python/paddle/tensor/manipulation.py]: unrelated

**Review by willthefrog — CHANGES_REQUESTED** [review]
remove unrelated changes

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/nn.py]: remove done

File: python/paddle/tensor/manipulation.py (hunk 2 above)
Comment (wangchaochaohu) [reply python/paddle/tensor/manipulation.py]: done
